tests/api/api/test_model_endpoints.py

```diff
@@ -31,40 +31,41 @@
 
 TEST_PROJECT = "test_model_endpoints"
 
+# Set a default v3io access key env variable
+v3io_access_key = "1111-2222-3333-4444"
```

Comment:
static variables should be named in all caps
```suggestion
V3IO_ACCESS_KEY = "1111-2222-3333-4444"
```